ocean-client/src/vault-maxi.ts

```diff
@@ -183,3 +184,17 @@ export async function main(event: maxiEvent,context: any): Promise<Object> {
         return { statusCode: 500 }
     }
 }
+
+if (typeof require !== 'undefined' && require.main === module) {
+    class contexTimer {
+        private start = Date.now();
+        public getRemainingTimeInMillis() { return 15000 * 60 - (Date.now() - this.start) } //on AWS max. 15 min execution time
+    }
+    process.env.VAULTMAXI_LOGID = process.env.VAULTMAXI_LOGID ?? "on " + os.hostname()
+    const myArgs = process.argv.slice(2);
+    var event = undefined;
+    if (!((myArgs.length>0) && (myArgs[0]=='run')))
+        event = { overrideSettings: undefined, checkSetup: true }
+    //@ts-ignore
+    main(event, new contexTimer)
```

Comment:
do we need this in the vault-maxi file or could we put it outside. 
so we don't have this code in the version that gets deployed to AWS? less risk that it does something weird (no idea if it does)

Reply:
The code is necessary to distinguish whether the module is called directly as a main module or imported from AWS by calling the function main(..). Similar to Python ' if __name_ == "__main__"'.

We can also make an extra file, e.g. app.ts, which imports vault-maxi and calls main(..). And 2 build-scripts. build_aws for vault-maxi.ts in target folder dist/aws and build_app for app.ts in dist/app.  In the folder out still remains the debugable version compiled with tsc.